packages/application-shell/src/components/setup-flop-flip-provider/setup-flop-flip-provider.tsx

```diff
@@ -141,17 +132,23 @@ export const SetupFlopFlipProvider = (props: Props) => {
           custom: getLaunchDarklyUserCustomFields(props.user, props.projectKey),
         },
       },
-      graphql: {
-        user: getGraphQlUser(props.user, props.projectKey),
-        uri: `${getMcApiUrl()}/graphql`,
-        query: AllFeaturesQuery,
-        pollingInteralMs: 1000 * 60 * 30, // 30 minutes
-        getRequestHeaders,
-        parseFlags,
+      http: {
+        user: getHttpUser(props.user, props.projectKey),
+        execute: async () => {
```

Comment:
@emmenko let me know what you think. This roughly would be the direction we talked about I think.

Reply:
Yep, looks much better now! 🎉